samples/scripts/models/Makefile

```diff
@@ -4,11 +4,12 @@ MLSERVER_FOLDER = mlserver_${MLSERVER_VERSION}
 TRITON_VERSION=$(shell grep 'nvidia/tritonserver' ../../../scheduler/Makefile | cut -d':' -f2 | cut -d'-' -f1 | tr '.' '-')
 TRITON_FOLDER=triton_${TRITON_VERSION}
 
+# note that income-lgb fails with new version of lightgbm
 .PHONY: train-all
-train-all: iris huggingface-text-gen-gpt2 huggingface-text-gen-tiny-stories moviesentiment income income-xgb income-lgb download-mnist-onnx download-cifar10-tensorflow wine-mlflow mnist-pytorch
```

Comment:
Is income-lgb no longer needed? (I see its corresponding make rule has remained in the Makefile)
lgtm otherwise.

Reply:
It is still part of the examples we provide, however building a new model with the new lgb library fails because of some api changes in the new library. Given that the old version still works then I thought we can fix this issue later in a follow up PR.